### DISCUSSION TITLE
version 1.9.0 installation

### DISCUSSION THREAD

**Discussion by brimb8** [hanai3Bi/Switch-OC-Suite#45] [Help | Troubleshooting]
It seems like the installation method has changed. I am on Atmosphere 1.6.2, Hekate 6.0.7, and copied the sys-clk-oc.7z to the root of my sd card. I cannot bring the memory clock up past 1600. In previous versions it seems there was a sdout card file, and having to move the kip file. Now I see a kip_patches folder, and not sure how to update the Hekate-ipl bootloader ini file now. Can you write an updated installation guide? Thanks!

**Comment by hanai3Bi** [comment]
your hekate_ipl.ini got overwritten while updating, so you need to put line again.

well process is same. you copy kip file, and copy sys-clk-oc files into sdcard.

**Reply by brimb8** [reply]
thanks, I have it working now!
